Dockerfile

```diff
@@ -23,13 +26,13 @@ RUN apk update &&\
     rm -f ${PACKAGE} SHA256SUMS &&\
     apk del wget tar --quiet &&\
     rm -rf /var/cache/apk/*
-
-# Note: we would prefer to use our own user to ensure the image does not run in root, but this comes with a lot of
-# limitations:
-# - difficulties with docker volume mounting
-# - will not work properly in gcp cloud run (especially with data mounting)
-# TODO: revist in future if limitations are still true
-# RUN addgroup -S mondoo && adduser -S -G mondoo mondoo
-# USER mondoo
+ 
 ENTRYPOINT [ "mondoo" ]
 CMD ["help"]
+
+# Rootless version of the container
+FROM root AS rootless
+
+RUN addgroup -S mondoo && adduser -S -G mondoo mondoo
```

Comment:
This creates uid 100:
```
/ $ id
uid=100(mondoo) gid=101(mondoo) groups=101(mondoo)
```

I think, they should be the same across the different images.

Reply:
I set the same uid and gid for the rootless ubi image